client/reader/sidebar/style.scss

```diff
@@ -112,3 +112,144 @@
 		}
 	}
 }
+
+.is-reader-page.theme-default .sidebar .sidebar__menu-link::after {
+	border-right-color: #fff;
+}
+
+.sidebar__menu {
+	.sidebar__menu-item-title {
+		display: flex;
+	}
+	.sidebar__heading,
+	.sidebar__menu-link {
+		/* Hover styles */
+		&:hover {
+			.sidebar__menu-icon.sidebar_svg-add {
```

Comment:
Do you think there's a way to do this without so many repeated stroke and fill declarations?

Reply:
Not really. This is the problem with applying custom styles to SVGs. Depending on the SVG, there are a variety of different configurations of path, g, gg, that might need to be edited compounded by the fact that sometimes you need to edit stroke and sometimes it's fill. I tried to bulk all of the SVGs with similar components together. There's still a bit of code to account for all of them.